sips/sip-x/sip-x-improved-stacking.md

```diff
@@ -0,0 +1,148 @@
+tokens are unlocked and cannot earn stacking rewards (Cooldown Cycle).
+
+The current PoX contract allows users to lock Stacks tokens during the prepare phase. This is too late for the next cycle
+and the tokens are locked without earning stacking rewards for one cycle.
+
+The cooldown cycle for unstacking from a pool presents a problem for network decentralization.
+As a stacker, when I delegate to a signer and that signer does not perform and gets low yield,
+I get penalized for switching. In today's model with cooldown cycles, users get double penalized
+if a validator does not perform. Even if there are penalties for signers not performing / being down,
+the switching costs are too high for users to switch (2 weeks worth of yield is a lot).
+
+## Proposed Solution
+
+A new PoX contract is proposed without
+the flow for solo stacking and moves responsibilities from pool operators to signers. It also
+defines the end of the locking period to be before the start of the stacking cycle.
+
+The user will be able to switch from one signer to another with a single contract call and without a cool down period.
+
+# Specification
+
+Applying these upgrades to the Stacks blockchain requires a consensus-breaking network upgrade,
+in this case, a hard fork. Like other such changes, this will require a new Stacks epoch.
+In this SIP, we will refer to this new epoch as Stacks 3.2.
+
+## Delegated Stacking only
+
+The following PoX contract functions shall be removed
+
+- stack-stx
+- stack-increase
+- stack-extend
+
+## Locking period
+
+Stacking transactions shall result in locking periods that end 100 blocks before the beginning of a stacking period. This includes
+
+- delegate-stack-stx
+- delegate-stack-extend
+
+The PoX contract has to be changed and return the unlock height as previous unlock height minus 100 blocks for mainnet. E.g.
+
+```
+(new-unlock-ht (- (reward-cycle-to-burn-height (+ u1 last-extend-cycle)) u100))
+```
+
+The unlocking process of the stacks node needs to be adapted accordingly.
+
+## Prepare phase
+
+The prepare phase shall start 10 blocks before the beginning of the stacking cycle and last for 10 blocks.
+
+The prepare cycle length defined in PoX contract has be changed to
+
+```
+(define-constant PREPARE_CYCLE_LENGTH (if is-in-mainnet u10 u5))
+```
+
+## Relationship Between Stackers and Signers
+
+The signature provided by a signers to a stackers shall use one of the following topics
+
+- delegate
+- delegate-increase
+- delegate-extend
+- agg-commit
+- agg-increase
+
+## Transition to PoX-5
+
+A new PoX contract requires that all stacked Stacks tokens are unlocked and Stackers need to lock their Stacks token again using the new PoX-5 contract. The process shall be similar to the previous upgrades of the PoX contract. PoX-4 contract shall be deactivated and PoX-5 contract shall be activated at the beginning of epoch 3.2 All locked Stacks tokens shall be unlocked automatically 1 block after the beginning of epoch 3.2.
+
```

Comment:
I think this section should be a bit more elaborate because it should cover potential complexities/risks related to sBTC and its dependence on PoX.

Reply:
sBTC is not involved here.